src/systemtest/java/com/ericsson/ei/systemtest/utils/PropertiesHandler.java

```diff
@@ -0,0 +1,59 @@
+package com.ericsson.ei.systemtest.utils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Map.Entry;
+import java.util.Properties;
+
+public class PropertiesHandler {
+
+    /**
+     * Function that set the properties.
+     */
+    public static void setProperties() throws Throwable {
+        final String eiConfigPropertiesFilepath;
+
+        if (System.getProperty("ei.config.properties.file.path") != null) {
+            eiConfigPropertiesFilepath = System.getProperty("ei.config.properties.file.path");
+        } else {
+            eiConfigPropertiesFilepath = "src/test/resources/system_test.properties";
```

Comment:
if this is only for system tests then it should be placed in src/systemtest/resources